src/pages/Basket/components/Summary/components/Order/components/Total/Total.ts

```diff
@@ -0,0 +1,44 @@
+import { Button } from 'components/Button'
+
+import { Text } from 'components/Text'
+
+import { CURRENCY } from 'enums'
+import { OrderForm } from 'forms'
+import {
+  createElementWithClassName,
+  getPricesByDiscount,
+  localStorageInstanse,
+  localStoragePromoCodeInstanse,
+} from 'helpers'
+
+import { ModalWindow } from './../../../../../../../../components/ModalWindow/ModalWindow'
```

Comment:
I guess my plugin or my teammate's plugin use this path and we didn't notice it
`import { ModalWindow } from 'components/ModalWindow'` - it will work